_rules/table-header-cells-refer-to-data-cells-d0f69e.md

```diff
@@ -0,0 +1,238 @@
+
+**Note:** Assigning headers cells to data cell is performed as per algorithm - [Forming relationships between data cells and header cells](https://html.spec.whatwg.org/multipage/tables.html#header-and-data-cell-semantics).
+
+## Assumptions
+
+_There are currently no assumptions._
+
+## Accessibility Support
+
+- `headers` attribute may not be consistently announced by assistive technologies.
+
+## Background
+
+- [Understanding Success Criterion 1.3.1: Information and relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
+- [H43: Using id and headers attributes to associate data cells with header cells in data tables](https://www.w3.org/WAI/WCAG21/Techniques/html/H43)
+- [Forming relationships between data cells and header cells](https://html.spec.whatwg.org/multipage/tables.html#header-and-data-cell-semantics)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The column header element has corresponding cell, within the same `table` element.
+
+```html
+<table>
+	<tr>
+		<th>Time</th>
+	</tr>
+	<tr>
+		<td>05:41</td>
+	</tr>
+</table>
+```
+
+#### Passed Example 2
+
+Each column header element has corresponding cells, within the same `table` element.
+
+```html
+<div role="table">
+	<div role="rowgroup">
+		<div role="row">
+			<span role="columnheader">Month</span>
+			<span role="columnheader">Top Temperature</span>
+		</div>
+	</div>
+	<div role="rowgroup">
+		<div role="row">
+			<span role="cell">July</span>
+			<span role="cell">40 C</span>
+		</div>
+		<div role="row">
+			<span role="cell">August</span>
+			<span role="cell">45 C</span>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed Example 3
+
+Each column header element has corresponding cells within the same table element. In this example the column headers have cells that span multiple columns.
+
+```html
+<table>
+	<thead>
+		<tr>
+			<th>Projects</th>
+			<th>Exams</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td colspan="2">15%</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+#### Passed Example 4
+
+Each row and column header element has corresponding cells, within the same element having a [semantic role](#semantic-role) of `grid`.
+
+```html
+<table role="grid">
+	<thead role="rowgroup">
+		<tr role="row">
+			<td></td>
+			<th role="columnheader">Breakfast</th>
+			<th role="columnheader">Lunch</th>
+			<th role="columnheader">Dinner</th>
+		</tr>
+	</thead>
+	<tbody role="rowgroup">
+		<tr role="row">
+			<th scope="row" role="rowheader">Day 1</th>
+			<td>8:00</td>
+			<td>13:00</td>
+			<td>18:00</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+#### Passed Example 5
+
+Each column header has corresponding cells. Usage of `headers` attribute does not change relationship between column headers and cells.
```

Comment:
Did you really meant "does not change relationship"? It should be "changes the relationship"